app/controllers/thread_controller.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+    class ThreadController extends AppController
+    {
+        public function index()
+        {
+            $threads = Thread::getAll();
+            $this->set(get_defined_vars());
+        }
+
+        public function view()
+        {
+            $thread = Thread::get(Param::get('thread_id'));
+            $comments = $thread->getComments();
+            $this->set(get_defined_vars());
+        }
+
+        public function write()
+        {
+            $thread = Thread::get(Param::get('thread_id'));
+            $comment = new Comment;
+            $page = Param::get('page_next', 'write');
+            switch ($page) {
+                case 'write':
+                    break;
+                case 'write_end':
+                    $comment->username = Param::get('username');
+                    $comment->body = Param::get('body');
+                    try {
+                        $thread->write($comment);
```

Comment:
Based from this [comment](https://github.com/floresjann/DietCake-Board/pull/1/files#r122123639) po, you can now do this
from line `20` modify :
```php
$page = Param::get('page_next', 'write');
if ($page == 'write_end') {
    $params = array(
        'thread_id' => Param::get('thread_id'),
        'username' => Param::get('username'),
        'body' => Param::get('body'),
    );
    try {
        Comment::create($params);
    } catch(ValidationException $e) {
        $page = 'write';
    }
}